pkgs/development/compilers/swift/default.nix

```diff
@@ -340,7 +340,7 @@ stdenv.mkDerivation {
     # TODO: consider using stress-tester and integration-test.
 
     # Match the wrapped version of Swift to be installed.
-    export LIBRARY_PATH=${icu}/lib:${libgcc}/lib:${libuuid.out}/lib:$l
+    export LIBRARY_PATH=${icu}/lib:${libgcc}/lib:${libuuid.lib}/lib:$l
```

Comment:
Could you switch in all instances to `lib.makeLibraryPath`? This will also make sure `lib` is used if available.

Reply:
Certainly! Done.